packages/expo/src/takeSnapshotAsync.ts

```diff
@@ -12,9 +12,13 @@ type SnapshotOptions = {
   result: "tmpfile" | "base64" | "data-uri" | "zip-base64";
 };
 
-export default async function takeSnapshotAsync(
-  node: ReactNativeNodeHandle | React.Component,
+export default async function takeSnapshotAsync<T>(
+  node: ReactNativeNodeHandle | React.Component | React.RefObject<T>,
```

Comment:
Agree that `React.RefObject<any>` would work fine but I think it is easier to remove the generic type parameter (`T`) in the future than to add it back. Either seems fine to me though.